src/CPE/public/cpe_dict.h

```diff
@@ -522,6 +522,9 @@ bool cpe_item_add_title(struct cpe_item *item, struct oscap_text *new_title);
 OSCAP_DEPRECATED(bool cpe_item_add_note(struct cpe_item *item, struct oscap_text *new_title));
 
 /// @memberof cpe_dict_model
+bool cpe_dict_model_add_item(struct cpe_dict_model *dict, struct cpe_item *new_item);
```

Comment:
The difference between OSCAP_IGETTER_GEN and OSCAP_IGETINS_GEN is that IGETINS also implements an insertion function (I'm assuming the INS stands for insert).  cpe_dict_model_add_item() is that insertion function.  It's just buried deep in a macro, but the prototype in the public header doesn't use the macro so it's easier for to read.  I didn't come up with this, I'm just following the convention that all the other functions were using in the module.